app/_kong_plugins/rate-limiting/index.md

```diff
@@ -15,17 +15,7 @@ If the underlying service or route has no authentication layer,
 the **Client IP** address is used. Otherwise, the consumer is used if an
 authentication plugin has been configured.
  
-The advanced version of this plugin, [Rate Limiting Advanced](https://docs.konghq.com/hub/kong-inc/rate-limiting-advanced/),
```

Comment:
Why get rid of references to the RLA plugin? We definitely want to call out the existence of a more advanced version of the plugin here.